interface/patient_file/ccr_import.php

```diff
@@ -22,39 +22,34 @@
  * @author  Ajil P M <ajilpm@zhservices.com>
  * @link    http://www.open-emr.org
  */
+require_once(dirname(__FILE__) . "../../globals.php");
 
-
-
-require_once(dirname(__FILE__) . "/../globals.php");
+use OpenEMR\Core\Header;
 
 ?>
 <html>
 <head>
+<?php Header::setupHeader(['opener']); ?>
```

Comment:
I should of removed that html_header_show; I'll do that this evening.

The asset manager is new and with no good documentation yet. Best way to get an idea of how it works is looking at comments in https://github.com/openemr/openemr/blob/master/library/core/src/Header.php and the "options" that can pick can be found here: https://github.com/openemr/openemr/blob/master/config/config.yaml
(note the ones with the autoload flag on are always loaded)

can get an idea of options.inc.php by playing around with Administration->layouts in openemr. The options.inc.php is basically there to support all the different datatypes. this is also used as the backbone for the layout based forms engine.

globals.php is the glue that holds everything together and is always run at top of each script